pr-preview/action.yml

```
@@ -0,0 +1,92 @@
+name: Pr preview
+description: "Build the image and deploy the pr code for preview"
+inputs:
+  ali_registry_info:
+    description: "Ali container registry information"
+    required: true
+  values_file:
+    description: "Local values yaml file"
+    required: false
+    default: ./.deploy/values.local.yaml 
+  image_tag_key_path:
+    description: "The image tag yaml key path"
+    required: false
+    default: "image.tag"
+  host_key_path:
+    description: "The application host yaml key path"
+    required: false
+    default: "applicationHosts"
+  chart_name:
+    description: "It is the name of your chart"
+    required: false
+    default: "application-chart-template"
```

这个参数我觉得可以设置为完整的值，而不是后半部分
```suggestion
  chart:
    description: "The chart reference, see: https://helm.sh/docs/helm/helm_install/#:~:text=com/charts/nginx-,CHART%20REFERENCES,-A%20chart%20reference"
    required: false
    default: "app-template/application-chart-template"
```